test/AsmJs/uint.js

```diff
@@ -12,10 +12,9 @@ let m = function (stdlib, foreign, heap) {
   }
   return f;
 }({}, {}, asmHeap);
-print(m(4294967295));
-print(m(4294967295));
 
-m = function (stdlib, foreign, heap) {
+
+m2 = function (stdlib, foreign, heap) {
```

Comment:
For some reason, the Compiler and execution messages got re-ordered after this change (but execution output remains correct). This happens consistently with rl.exe, not when run from command-line. The new output is:
```
Successfully compiled asm.js code
0
0
Successfully compiled asm.js code
1
1
```

I had to re-order the test to get the original baseline output:
```
Successfully compiled asm.js code
Successfully compiled asm.js code
0
0
1
1
```
@MikeHolman , any idea what might be going on here ? And if it could be an issue ?